database/controllers/dbm.js

```diff
@@ -206,8 +222,18 @@ module.exports = {
     //Descriptions:  This function will take a user's badge # and return
     // a JSON object with all the stations the user has been trained on and
     // can use.
-    getPrivileges(bId){
-        return true;
+    getPriv(bId){
+        return privileges.findAll({
```

Comment:
I dont think you need two functions in this case. Pretty sure you can just do: `var results = await privileges.findAll(...)`. I dont think you need to use `async`. Then results will contain what you need and you can do the checks on that.

Reply:
Ah, okay that makes sense.  I'm taking off for the day, but can have it fixed by tomorrow morning.

Reply:
Sounds good, thanks!

Reply:
Everything should be good now.